integration/update_compat_test.go

```diff
@@ -485,3 +485,37 @@ func TestUpdateCompatMultiFlagCommand(t *testing.T) {
 
 	testUpdateCommandCompat(t, testCases)
 }
+
+func TestReplaceKeepOrderCompat(t *testing.T) {
+	setup.SkipForTigrisWithReason(t, "Schema violation")
```

Comment:
Minor comment. We don't need to skip tigris here. The violation here comes from using not string `_id` and using random provider.

I think we can make sure tigris schema is not violated here by selecting provider. For example, `shareddata.Int32s` provider would work for your int data if we update tigris schema.
```
r := setup.SetupCompatWithOpts(t, &setup.SetupCompatOpts{Providers: []shareddata.Provider{shareddata.Int32s}})
```
Adding schema for`a`, `b`, `c` would be enough. `foo` is already there though.
https://github.com/FerretDB/FerretDB/blob/458bc77112ad35fb1653b989e3cfba1b96451d28/integration/shareddata/scalars.go#L377
```
"a": {%%type%%},
"b": {%%type%%},
"c": {%%type%%},
```

And `_id`s have to be string to not violate tigris schema we set in collections :sweat_smile: